packages/p2p/src/utils/websocket.js

```diff
@@ -71,16 +71,19 @@ const getModifiedP2POfferList = response => {
             price_rate,
             transaction_currency,
             transaction_currency_decimals,
-            advertiser_name: filtered_list[i].agent_name,
-            advertiser_notes: filtered_list[i].offer_description,
+            advertiser_name: ObjectUtils.getPropertyValue(filtered_list[i], ['advertiser_details', 'name']),
+            advertiser_id: ObjectUtils.getPropertyValue(filtered_list[i], ['advertiser_details', 'id']),
+            advertiser_notes: filtered_list[i].description,
             display_available_amount: formatMoney(offer_currency, available_amount),
             display_max_available: formatMoney(offer_currency, max_available), // for displaying limit fields in buy/sell and ads table
             display_min_available: formatMoney(offer_currency, min_available), // for displaying limit fields in buy/sell and ads table
             display_offer_amount: formatMoney(offer_currency, offer_amount),
             display_payment_method: map_payment_method[payment_method] || payment_method,
             display_price_rate: formatMoney(transaction_currency, price_rate),
-            offer_id: filtered_list[i].offer_id,
-            type: filtered_list[i].type,
+            id: filtered_list[i].id,
+            // for view in my ads tab, we should show the real type of the ad
+            // for view in buy/sell table, we should show flip the ad type
+            type: is_from_my_ads ? filtered_list[i].type : filtered_list[i].counterparty_type,
```

Comment:
Can we rename this boolean variable to something like `is_original_type`, or `is_of_advert_placed_type`?